docs/src/main/sphinx/sql/merge.md

```diff
@@ -73,12 +77,11 @@ MERGE INTO accounts t USING monthly_accounts_update s
 ```
 
 `MERGE` into the target table from the source table, deleting any matching
-target row for which the source address is Centreville.  For all other
-matching rows, add the source purchases and set the address to the source
-address, if there is no match in the target table, insert the source
-table row:
+target row for which the source address is `Centreville`. For all other matching
+rows, add the source purchases and set the address to the source address, if
```

Comment:
Maybe better to end the sentence after the comma, e.g.:

rows, add the source purchases and set the address to the source address. If

Reply:
yeah ... good point